### DISCUSSION TITLE
Minion skill with 0 weight

### DISCUSSION THREAD

**Discussion by bernardo7894** [xanthics/PoE_Weighted_Search#31] [Q&A] [answered]
I realize that it says that "If you are using a minion skill as primary and see all 0 weights, try using FullDPS in PoB first". However, I don't know what that means. Is it an option in PoB? Thanks.

**Comment by bernardo7894** [comment, marked as answer]
Well, I figured it out. Took me a while to find it.

![2021-08-01_19-43-43](https://user-images.githubusercontent.com/3051242/127781929-076f8aaa-0faf-40b5-a36b-e12c2463fa77.png)

I don't think I can delete the question itself, so I went ahead and answered it myself.

**Comment by codeninja** [comment]
@bernardo7894 's suggestion isn't working for me.

![image](https://user-images.githubusercontent.com/14914/169867397-334ec464-e7f6-4cfb-856c-f0bf82f917ca.png)


I get all 0's on the weights :(
